src/System.Private.CoreLib/src/System/Diagnostics/Eventing/EventPipeController.cs

```diff
@@ -50,37 +49,30 @@ internal sealed class EventPipeController
         private const string ConfigKey_MultiFileSec = "MultiFileSec";
 
         // The default set of providers/keywords/levels.  Used if an alternative configuration is not specified.
-        private static readonly EventPipeProviderConfiguration[] DefaultProviderConfiguration = new EventPipeProviderConfiguration[]
+        private static EventPipeProviderConfiguration[] DefaultProviderConfiguration => new EventPipeProviderConfiguration[]
         {
             new EventPipeProviderConfiguration("Microsoft-Windows-DotNETRuntime", 0x4c14fccbd, 5, null),
             new EventPipeProviderConfiguration("Microsoft-Windows-DotNETRuntimePrivate", 0x4002000b, 5, null),
             new EventPipeProviderConfiguration("Microsoft-DotNETCore-SampleProfiler", 0x0, 5, null),
         };
 
         // Singleton controller instance.
-        private static EventPipeController s_controllerInstance = null;
-
-        // Initialization flag used to avoid initializing FrameworkEventSource on the startup path.
-        private static bool s_initializing = false;
+        private static EventPipeController s_controllerInstance;
```

Comment:
@jkotas, is this a safe change for CoreLib where we don't initialize locals?  That is the reason for some of this, but I could have definitely overdone it.

Reply:
statics != locals. Statics are always zero-initialized.

Also, skipping the initialization of locals only affects unsafe code.

Reply:
object types are always cleared regardless of initlocals; else they will report junk data locations to the GC

Reply:
Great.  Thanks much.